firefox_ui_tests/functional/private_browsing/test_about_private_browsing.py

```diff
@@ -2,65 +2,54 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from marionette_driver import By
+from marionette_driver import By, Wait
 
 from firefox_ui_harness.decorators import skip_if_e10s
 from firefox_ui_harness import FirefoxTestCase
 
+from firefox_puppeteer.ui.windows import BrowserWindow
+
 
 class TestAboutPrivateBrowsing(FirefoxTestCase):
 
     def setUp(self):
         FirefoxTestCase.setUp(self)
-        self.pb_url = self.marionette.absolute_url('private_browsing/about.html?')
 
-    def tearDown(self):
-        self.marionette.close()
+        self.support_url = 'about:blank?'
+        self.pb_url = self.support_url + 'private-browsing'
+
+        self.prefs.set_pref('app.support.baseURL', self.support_url)
 
     @skip_if_e10s
     def testCheckAboutPrivateBrowsing(self):
         self.assertFalse(self.browser.is_private)
 
-        self.prefs.set_pref('app.support.baseURL', self.pb_url)
-
         with self.marionette.using_context('content'):
             self.marionette.navigate('about:privatebrowsing')
 
-            description = self.browser.get_entity(
-                'aboutPrivateBrowsing.subtitle.normal')
-
-            status_node = self.marionette.find_element(By.CSS_SELECTOR,
-                                                       'p.showNormal')
-            self.assertEqual(status_node.text, description)
-
-            access_key = self.browser.get_entity(
-                'privatebrowsingpage.openPrivateWindow.accesskey')
-
-            # Send keys to the top html element.
-            top_html = self.marionette.find_element(By.TAG_NAME, 'html')
-            top_html.send_keys(self.keys.SHIFT, self.keys.ACCEL, access_key)
-
-        self.wait_for_condition(lambda mn: len(self.windows.all) == 2)
-        self.browser_pb = self.windows.switch_to(lambda win: win.is_private)
-        self.assertTrue(self.browser_pb.is_private)
+            status_node = self.marionette.find_element(By.CSS_SELECTOR, 'p.showNormal')
+            self.assertEqual(status_node.text,
+                             self.browser.get_entity('aboutPrivateBrowsing.subtitle.normal'),
+                             'Status text indicates we are in private browsing mode')
```

Comment:
This tests the localized text indirectly, but yes we cannot figure out if it is localized or not. We can also not check if the localized text has been correctly translated.